core/io/resource_loader.h

```diff
@@ -254,6 +257,19 @@ class ResourceLoader {
 	static void set_abort_on_missing_resources(bool p_abort) { abort_on_missing_resource = p_abort; }
 	static bool get_abort_on_missing_resources() { return abort_on_missing_resource; }
 
+	static void set_reimport_file_on_failed_load(bool p_reimport_file_on_failed_load) { reimport_file_on_failed_load = p_reimport_file_on_failed_load; }
+	static bool get_reimport_file_on_failed_load() { return reimport_file_on_failed_load; }
+	static void set_reimport_file_func(ResourceLoaderImportFile p_import_file) {
+		import_file = p_import_file;
+	}
+	static Error reimport_file(const String &p_path, bool p_update_file_system) {
```

Comment:
Unused.

Reply:
I removed the reimport_file method and added a check in `ResourceFormatImporter::load` to be sure `ResourceLoader::import_file` is not null before calling it.